src/realm/table_view.hpp

```diff
@@ -330,13 +362,6 @@ class TableView : public ObjList {
     // Sort m_key_values according to multiple columns
     void sort(SortDescriptor order);
 
-    // Get the number of total results which have been filtered out because a number of "LIMIT" operations have
```

Comment:
I believe this was used in old query based sync as an attempt at a poor man's pagination of results. That feature will need a better design anyways so good riddance to this.